app/services/bulk_upload/sales/year2024/row_parser.rb

```diff
@@ -1450,6 +1450,8 @@ def validate_buyer1_economic_status
   end
 
   def validate_buyer2_economic_status
```

Comment:
I looked at getting rid of this entirely - it is covered by the validations on the model, but the error messages you get are then a bit confusing so I went for this.